merge/conflict_test.go

```diff
@@ -61,3 +61,34 @@ conflicts with "Bob":
 		t.Errorf("Got %v, wanted %v", got.Error(), wanted)
 	}
 }
+
+func TestToSet(t *testing.T) {
```

Comment:
Thanks for adding the test, now that you've done that, I realize that it's aggregating the conflicts one has with all users. You need to document the method above to explain that.

Reply:
Ah yes, done